### DISCUSSION TITLE
Type error on src. What am I doing wrong?

### DISCUSSION THREAD

**Discussion by rudy128** [muxinc/next-video#271] [Q&A]
`
import Video from 'next-video'
import React from 'react'
import adamVid from './../../videos/Use_your_Video_with_Top_8_Doodles_Transition_FX_Green_Screen____by_Green_Pedia.mp4.json';

const P1Video = () => {
    return (
        <Video className="w-full absolute -z-50 h-full" test={true} autoPlay loop muted controls={false} playsInline src={adamVid} />

    )
}

export default P1Video
`

![image](https://github.com/muxinc/next-video/assets/77375030/4812ddf2-bdad-4b97-bf6f-31f6ae38c136)

Can't figure out the problem

**Comment by luwes** [comment]
the `mp4.json` should be created automatically. 
try removing the `.json` from the import url and if you haven't already run `npx next-video sync` to upload it to one of the video providers
